index.js

```diff
@@ -201,34 +243,29 @@ module.exports = {
           live({ old: true }),
           toPullStream()
         ),
+        pull.filter((msg) =>
+          lodashGet(msg, 'value.content.recps', []).includes(ssb.id)
+        ),
         pull.asyncMap((msg, cb) => {
-          // TODO: call ssb-db2 reindexEncrypted
```

Comment:
we should add tests for this

Reply:
Maybe I should just remove reindexEncrypted from this PR then.

Reply:
yeah we have an issue for this, maybe better to do it all later https://github.com/ssbc/ssb-tribes2/issues/17